shop_project/shop_app/views.py

```diff
@@ -1,11 +1,37 @@
 from django.shortcuts import render
 from shop_app.models import Client, Product, Order
 import datetime
+from forms import ProductForm
+from flask import redirect
+from django.core.files.storage import FileSystemStorage
+
 
 start_time = datetime.datetime(2023, 5, 2, 12, 0, 0)
 current_time = datetime.datetime.now()
 time_passed = current_time - start_time
 
+
+def product_edit(request, product_id):
+    product = Product.objects.get(id=product_id)
+    if request.method == "POST":
+        form = ProductForm(request.POST, request.FILES, instance=product)
+        if form.is_valid():
+            data = form.cleaned_data
+            Product.objects.update_or_create(
+                name = data['name'],
+                description = data['description'],
+                price = data['price'],
+                number = data['number'],
+                product_added = data['product_added'],
+            )
+            image = form.cleaned_data['image']
+            fs = FileSystemStorage()
+            fs.save(image.name, image)
+            return redirect("product_item")            
+    else:
+        form = ProductForm()
```

Comment:
Чтобы поля были заполнены, когда подгружается уже существующий продукт, тогда тут нужно указать
`instance=product`
```suggestion
        form = ProductForm(instance=product)
```